index.html

```diff
@@ -382,7 +383,8 @@ <h5><b>Here's our tutorial covering these methods:</b></h5>
                               (the website allows either sharing a link for joining or manually approving new participants).
                               This allows you to screen the peers,
                               acknowledge their contributions (e.g., make a leaderboard), and
-                              ban accounts who behave maliciously.
+                              ban accounts who behave maliciously. You will need to use <a href="https://collaborative-training-auth.huggingface.co/docs">authentication system</a> or deploy your own for this system. 
+                              Our <a href="https://github.com/huggingface/collaborative-training-auth/tree/demo-neurips">server implementation</a> might be a good start.
```

Comment:
This is maybe not the good place to had it, but this piece of code might be useful for some future experiments :slightly_smiling_face: